srcpkgs/python3-parso/template

```diff
@@ -1,6 +1,7 @@
 # Template file for 'python3-parso'
 pkgname=python3-parso
-version=0.8.0
+reverts="0.8.0_1"
+version=0.7.1
 revision=1
 archs=noarch
```

Comment:
We're deprecating `noarch`, please drop it.
Also, bump `revision` please.

Reply:
Anyway, I fixed it for you. Build farm is busy for now, so no rush to push this out, waiting for @ahesford  instead.

Reply:
When updating parso, I looked at jedi but didn't catch the breaking changed introduced in `PosixPath`; I mistakenly assumed the only critical change was a drop in py2 support, which wouldn't matter since we only build it for py3. Sorry for the mistake. The jedi maintainer keeps promising an 0.18.0 release "in a matter of weeks", but it seems to have been delayed by several weeks, so this is the right move now.

Also, `revision=1` is appropriate here. There was never a `python3-parso-0.7.1_1` because I dropped the py3 subpackage from `python-parso` when moving from 0.7.0 to 0.7.1.

Reply:
OK, I push this out, hope you don't mind.

Reply:
Looks good from my end! Thanks for getting this merged.